server/api.go

```diff
@@ -122,6 +129,11 @@ type AIBotInfo struct {
 	UserIDs            []string               `json:"userIDs"`
 }
 
+type AIBotsResponse struct {
+	Bots          []AIBotInfo `json:"bots"`
+	SearchEnabled bool        `json:"searchEnabled"`
```

Comment:
I don't know if it makes sense, but maybe we want to make search enabled a setting for the bot itself. That way you can decide what AIs access the search and what is doesn't, increasing the granularity of the control and permissions over the functionality. This bot can search, and is available to this users, this bot is available to everybody but it can't search. Maybe makes no sense, but it is just a though. Feel free to ignore this comment :)

Reply:
Agree that it might be useful. Same as being able to control what specific tools are available to which bots. I think a UI for determining which tools each bot can access is inevitable. Going to defer this from this PR for now and add this as a control for when we implement a unified UI for all tools.